<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Configuration</class>
 <widget class="QDialog" name="Configuration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>548</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>290</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::SaveAll</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>361</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>CameraCalibrate</string>
   </property>
   <widget class="QPushButton" name="ProduceMatrix">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>25</y>
      <width>150</width>
      <height>75</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>75</height>
     </size>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;生成畸变校准矩阵&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Produce Matrix</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ParameterContrast">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>100</y>
      <width>150</width>
      <height>75</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>75</height>
     </size>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;畸变校准测试&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Calibrate Contrast</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Picture_N">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>24</y>
      <width>39</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>62</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>65</y>
      <width>108</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>P_CornerNumber_col</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>151</y>
      <width>108</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Accuracy</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>43</y>
      <width>108</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>P_CornerNumber_row</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>108</y>
      <width>108</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>CellSize_height</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="P_CornerNumber_col">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>66</y>
      <width>39</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Accuracy">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>150</y>
      <width>39</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0.01</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>129</y>
      <width>108</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Iteration_N</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Iteration_N">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>129</y>
      <width>39</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>50</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="CellSize_height">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>108</y>
      <width>39</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>20</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="P_CornerNumber_row">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>45</y>
      <width>39</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>86</y>
      <width>108</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>CellSize_width</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>22</y>
      <width>108</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Picture_N</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="CellSize_width">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>87</y>
      <width>39</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>22</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>271</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>CameraSetting</string>
   </property>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>66</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Camera gain</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="Brightness_spinBoxdouble">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>54</width>
      <height>20</height>
     </rect>
    </property>
    <property name="decimals">
     <number>0</number>
    </property>
    <property name="value">
     <double>50.000000000000000</double>
    </property>
   </widget>
   <widget class="QSlider" name="CameraGama">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>80</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>40</number>
    </property>
    <property name="pageStep">
     <number>5</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>66</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Gama</string>
    </property>
   </widget>
   <widget class="QSlider" name="CameraGain">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>300</number>
    </property>
    <property name="value">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>66</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>brightness </string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="Cameragain_spinBoxdouble">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>50</y>
      <width>54</width>
      <height>20</height>
     </rect>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="maximum">
     <double>30.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="Gama_spinBoxdouble">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>80</y>
      <width>54</width>
      <height>20</height>
     </rect>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="maximum">
     <double>4.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="stepType">
     <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
    </property>
    <property name="value">
     <double>0.500000000000000</double>
    </property>
   </widget>
   <widget class="QSlider" name="CameraBrightness">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>101</width>
      <height>22</height>
     </size>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>500</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>141</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>ScannerSetting</string>
   </property>
   <widget class="QLabel" name="Focallabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>focal</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pixelSizeLine">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>39</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0.009</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="RGBlabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>154</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>rgb</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="baseLineLine">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>76</y>
      <width>39</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>100</string>
    </property>
   </widget>
   <widget class="QLabel" name="baselinelabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>76</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>baseLine</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="laserAngleLine">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>128</y>
      <width>39</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>70</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="focalLine">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>24</y>
      <width>39</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>2.7</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="rgeLine">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>154</y>
      <width>39</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>100</string>
    </property>
   </widget>
   <widget class="QLabel" name="laserAnglelabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>128</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>laserAngle</string>
    </property>
   </widget>
   <widget class="QLabel" name="stepangleplable">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>102</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>stepAngle</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="stepAngleLine">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>102</y>
      <width>39</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0.7</string>
    </property>
   </widget>
   <widget class="QLabel" name="PixelSizelable">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>24</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>pixelSize</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Configuration</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Configuration</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
